ledger/participant-state/kvutils/tools/integrity-check/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/RawPreExecutingCommitStrategySupport.scala

```diff
@@ -10,16 +10,12 @@ import akka.stream.Materializer
 import com.daml.ledger.on.memory.{InMemoryLedgerStateAccess, InMemoryState, Index}
 import com.daml.ledger.participant.state.kvutils.DamlKvutils.DamlStateValue
 import com.daml.ledger.participant.state.kvutils.{KeyValueCommitting, export}
-import com.daml.ledger.validator.preexecution.{
-  EqualityBasedPostExecutionConflictDetector,
-  PreExecutingSubmissionValidator,
-  PreExecutingValidatingCommitter,
-  RawKeyValuePairsWithLogEntry,
-  RawPostExecutionWriter,
-  RawPreExecutingCommitStrategy,
-  TimeBasedWriteSetSelector,
+import com.daml.ledger.validator.preexecution._
```

Comment:
I'd prefer to avoid wildcard imports if possible. Could you adjust your IntelliJ settings so that it doesn't do this please?